RunningOrder/Story/StoryRow.swift

```diff
@@ -9,17 +9,21 @@
 import SwiftUI
 
 struct StoryRow: View {
```

Comment:
this struct should take the story in readonly. No need of the manager here isn't it ?